frontend/src/styles/LoginRegistrationPage.css

```diff
@@ -1,97 +1,246 @@
-div.loginHeader {
-  font-size: 40px;
-  color: #1DE28F;
-}
+@import url('https://fonts.googleapis.com/css?family=Teko');
 
+:root {
+  --BackgroundColor: #202020;
+  --themeColor: #1DE28F;
+  --bodyColor: var(--baseColor);
+  --FontFamily: 'Teko';
+  --FontSize: 14px;
+  --FontWeight: 300;
+  --LineHeight: 1.5;
+}
+body {
+  background-color: var(--BackgroundColor);
+  color: var(--bodyColor);
+  font-family: var(--FontFamily);
+  font-size: var(--FontSize);
+  font-weight: var(--FontWeight);
+  height: 100%;
+  line-height: var(--LineHeight);
+  margin: 0;
+  min-height: 60px;
+}
+form{
+  padding-top: 60px;
+}
 div.login1-button {
+  text-align: center;
+  opacity: 0.7;
+  cursor: pointer;
+  font-family: var(--FontFamily);
   width: 45px;
+  font-size: 34px;
+  letter-spacing: 2px;
   float: none;
   margin: 0 auto;
-  margin-top: 5px;
+  margin-top: 10px;
   padding: 5px;
-  color: #1DE28F;
+  padding-top: 20px;
+  color:var(--themeColor);
   font-weight: bold;
-  cursor: pointer;
 }
-
-div.login2-button {
+div.login1-button:hover {
```

Comment:
Again, I dont think login1-button is being used. Just verify. If it's not being used, remove this block of code.

Reply:
I don't think this is used anywhere, please remove.